lib/adoptoposs_web/templates/layout/root.html.eex

```diff
@@ -30,5 +30,12 @@
     <footer class="bg-gray-900 text-red-400 py-6 md:py-10">
       <%= render AdoptopossWeb.SharedView, "footer.html", assigns %>
     </footer>
+
+    <div class="w-full sticky bottom-0 bg-gray-900 flex items-center justify-center py-4 cookies-box hidden" id="cookies-banner">
+      <p class="text-white text-center m-0">By using this website you agree to our <a href="/privacy">Cookie Policy</a>.</p>
+      <a id="close-cookies-banner" class="absolute w-auto right-0 h-3 w-3 cursor-pointer">
+        <svg xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:svgjs="http://svgjs.com/svgjs" version="1.1" width="15" height="15" x="0" y="0" viewBox="0 0 496.096 496.096" style="enable-background:new 0 0 512 512" xml:space="preserve" class=""><g> <g xmlns="http://www.w3.org/2000/svg"> <g> <path d="M259.41,247.998L493.754,13.654c3.123-3.124,3.123-8.188,0-11.312c-3.124-3.123-8.188-3.123-11.312,0L248.098,236.686    L13.754,2.342C10.576-0.727,5.512-0.639,2.442,2.539c-2.994,3.1-2.994,8.015,0,11.115l234.344,234.344L2.442,482.342    c-3.178,3.07-3.266,8.134-0.196,11.312s8.134,3.266,11.312,0.196c0.067-0.064,0.132-0.13,0.196-0.196L248.098,259.31    l234.344,234.344c3.178,3.07,8.242,2.982,11.312-0.196c2.995-3.1,2.995-8.016,0-11.116L259.41,247.998z" fill="#ffffff" data-original="#000000" style="" class=""/></g></g></svg>
+      </a>
+    </div>
```

Comment:
Maybe you can move the cookie-banner into a seperate shared template, similar to what we did with `google_analytics.html.eex`? It might keep our root template a bit clearer.